lib/cubit/account/account_cubit.dart

```diff
@@ -278,6 +281,29 @@ class AccountCubit extends Cubit<AccountState> with HydratedMixin {
     });
   }
 
+  void _listenConnectivityChanges() {
+    connectivitySubscription = Connectivity().onConnectivityChanged.skip(1).listen(
+      (result) async {
+        _log.info("Connection changed to: $result");
+        bool hasNetworkConnection = !result.contains(ConnectivityResult.none);
+        if (!hasNetworkConnection) {
+          emit(state.copyWith(connectionStatus: ConnectionStatus.disconnected));
+        }
+
+        bool isDisconnected = state.connectionStatus == ConnectionStatus.disconnected;
+        if (hasNetworkConnection && isDisconnected) {
+          await connect();
```

Comment:
What happens if the SDK is successfully connected then, connectivity is offline then online. Do we create a new SDK instance?

Reply:
Moved away from this approach and network connectivity issues are handled separately on ConnectivityCubit.